FIRO_TSEnsembles/src/main/java/hec/TableCompression.java

```diff
@@ -8,12 +8,12 @@
 import java.util.zip.GZIPInputStream;
 import java.util.zip.GZIPOutputStream;
 
- public class EnsembleCompression {
+ public class TableCompression {
 
 
     /**
      * UnPack a GZIP a byte array
-     * returns an ensemble  (float[][])
+     * returns a Table  (float[][])
```

Comment:
returns an 2d array

Reply:
returns a 2d Array of float (float[rows][columns]) that is not jagged